db/migrate/20180331230509_add_loan_public_level_options.rb

```diff
@@ -0,0 +1,18 @@
+class AddLoanPublicLevelOptions < ActiveRecord::Migration[5.1]
+  def up
+    loan_public_level = OptionSet.find_or_create_by(division: Division.root, model_type: Loan.name,
+      model_attribute: 'public_level')
+    loan_public_level.options.destroy_all
```

Comment:
Why destroy instead of just updating the translations?

Reply:
I followed other migrations for option set seeds from the past.

How would you approach the updates?

Maybe I can check in with you today about this, if you have capacity.